bundle-size/github-utils.js

```diff
@@ -242,6 +236,73 @@ class GitHubUtils {
 
     return reviewer;
   }
+
+  /**
+   * Add an bundle size reviewer to the pull request.
+   *
+   * @param {!Octokit.PullsListReviewRequestsParams} pullRequest GitHub Pull
+   *   Request params.
+   * @param {!Array<string>} approverTeams list of all the teams whose members
+   * can approve the bundle-size change of this pull request.
+   */
+  async addBundleSizeReviewer(pullRequest, approverTeams) {
+    const requestedReviewersResponse = await this.github.pullRequests.listReviewRequests(
+      pullRequest
+    );
+    const reviewsResponse = await this.github.pullRequests.listReviews(
+      pullRequest
+    );
+    const existingReviewers = new Set([
+      ...requestedReviewersResponse.data.users.map(user => user.login),
+      ...reviewsResponse.data.map(review => review.user.login),
+    ]);
+
+    const potentialReviewers = await this.getTeamMembers_(approverTeams);
+    if (
+      potentialReviewers.length &&
+      potentialReviewers.every(
+        potentialReviewer => !existingReviewers.has(potentialReviewer)
+      )
```

Comment:
`[].some(...) ==> false`, so you can remove the length check and move the negation out to the top level. ie:
`if (!potentialReviewers.some(existingReviewers.has, existingReviewers))` (or the arrow-notation version).

Reply:
I got mixed up so much in the logic here that I ended up with this abomination above. Fixed.